### ISSUE TITLE
[FEATURE] Google search

### ISSUE THREAD

**Issue by KITT3000** [jtsage/FS19_Mod_Checker#30]
I use the DuckDuckGo search engine in my browser and it annoys me a bit as the program does not use my default but Google, it is possible to search in the default search engine, not one set in advance?

**Comment by jtsage** [comment]
Hmm.  This might be a bit hard.  It doesn't look right now like there is an OS hook to use the default search engine - only the default browser.  But I might be able to at least make the choice customizable - it'll depend on how/if I can save user preferences easily, as choosing a search engine every time you open the app would get real old real fast.

**Comment by jtsage** [comment]
So, yeah, I can't do anything to trigger the "default search engine".  But what I can do (and did) is add DuckDuckGo and Bing both to the context menu.  The only other one I was sort of considering was yandex, as it has much better coverage or sites in the Slavic languages - or at least, it did last time I messed with it, which isn't recent.

I did look at user preferences - and there are some good options, but none that don't add a pretty decent layer of complexity. I'll mark this as a "come back here" if I end up implementing more things that are going to be user specific.
